pkg/proto/pbgo/core/api.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.0
+// 	protoc-gen-go v1.34.2
```

Comment:
the update is on purpose to use the last bug fix release. but I could have use v1.34.0